src/pages/settings/AboutPage.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View, ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'lodash';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import CONST from '../../CONST';
+import {
+    Link, Eye, MoneyBag, Bug, NewWindow,
+} from '../../components/Icon/Expensicons';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import MenuItem from '../../components/MenuItem';
+import Logo from '../../../assets/images/expensify-cash.svg';
+import {version} from '../../../package.json';
+import openURLInNewTab from '../../libs/openURLInNewTab';
+import ONYXKEYS from '../../ONYXKEYS';
+
+const propTypes = {
+    /** Onyx Props */
+
+    /** All reports shared with the user */
+    reports: PropTypes.shape({
+        /** ID of the report */
+        reportID: PropTypes.number,
+
+        /** Name of the report */
+        reportName: PropTypes.string,
+    }).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const AboutPage = ({translate, reports}) => {
+    const menuItems = [
+        {
+            translationKey: 'initialSettingsPage.aboutPage.appDownloadLinks',
+            icon: Link,
+            action: () => { Navigation.navigate(ROUTES.SETTINGS_APP_DOWNLOAD_LINKS); },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.viewTheCode',
+            icon: Eye,
+            iconRight: NewWindow,
+            action: () => { openURLInNewTab(CONST.GITHUB_URL); },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.viewOpenJobs',
+            icon: MoneyBag,
+            iconRight: NewWindow,
+            action: () => { openURLInNewTab(CONST.UPWORK_URL); },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.reportABug',
+            icon: Bug,
+            action: () => {
+                const reportKey = _.findKey(reports, ['reportName', CONST.CONCIERGE_CHAT_NAME]);
+                if (reportKey) {
+                    const report = reports[reportKey];
+                    if (Array.isArray(report.participants)
+                        && report.participants.length === 1
+                        && report.participants[0] === CONST.EMAIL.CONCIERGE) {
+                        Navigation.navigate(ROUTES.getReportRoute(report.reportID));
+                    }
```

Comment:
You might be able to use this[ existing function](https://github.com/Expensify/Expensify.cash/blob/2a86ca3c2c82e06ee751805b864a18c02d2479a4/src/libs/actions/Report.js#L849) instead:
```
fetchOrCreateChatReport([currentUserEmail, CONST.EMAIL.CONCIERGE], navigate);
```

This is simpler, and should guarantee we launch the chat even if the report does not exist locally.

Reply:
Done